syntax/index.html

```diff
@@ -649,14 +651,20 @@
             Split the header and data rows into cells using the <a>delimiter</a>. Values that are enclosed within the <a>quote character</a> may contain the <a>delimiter</a>. The <a>quote character</a> may be escaped using the <a>escape character</a> where it appears within cells. If the <a>escape character</a> is not the same as the <a>quote character</a> then the <a>escape character</a> is also used to escape the character that immediately follows it.
           </p>
           <p>
-            If <a>trim</a> is <code>true</code> or <code>start</code> then whitespace from the start of values that are not enclosed must be removed from the value. If <a>trim</a> is <code>true</code> or <code>end</code> then whitespace from the end of values that are not enclosed must be removed from the value.
+            If <a>trim</a> is <code>true</code> or <code>start</code> then whitespace from the start of values that are not enclosed MUST be removed from the value. If <a>trim</a> is <code>true</code> or <code>end</code> then whitespace from the end of values that are not enclosed MUST be removed from the value.
           </p>
         </li>
+        <li>
+          Construct <a>embedded metadata</a> as Table Metadata (see <cite><a href="http://www.w3.org/TR/tabular-metadata/#tables"></a></cite> in [[!tabular-metadata]]) using the <code>@context</code> from <a>merged metadata</a> and setting the <code>language</code> annotation to the default language from the <code>@context</code>.</li>
         <li>In each row, ignore the number of columns indicated by the <a>skip columns</a> parameter. Always start from the first character in the row when counting columns (see <a href="#bidirectionality-in-csv-files" class="sectionRef"></a>).</li>
         <li>Gather the number of header columns indicated by the <a>header column count</a> parameter. Always start from the first character in the row when counting columns (see <a href="#bidirectionality-in-csv-files" class="sectionRef"></a>).</li>
-        <li>Each cell within a header row that is not in a skipped or header column is a <code>title</code> annotation on that column.</li>
+        <li>Each cell within a header row that is not in a skipped or header column is a <code>title</code> annotation on that column.
+          <div class="issue">How are titles from multiple header rows handled? Presume that multiple header cells in the same column are joined together with a newline separator.</div>
```

Comment:
Could go either way, but in my experience, multiple header lines typically represent multiple parts of a column title, rather than different titles. Anyway, it's just a corner case we need to document.